linux_os/guide/system/permissions/restrictions/enable_nx/install_PAE_kernel_on_x86-32.rule

```diff
@@ -13,8 +13,8 @@ description: |-
     be installed to enable XD or NX support:
     <pre>$ sudo yum install kernel-PAE</pre>
     The installation process should also have configured the
-    bootloader to load the new kernel at boot. Verify this at reboot
-    and modify <tt>/etc/default/grub</tt> if necessary.
```

Comment:
@cipherboy this should actually stay.

Reply:
@redhatrises :) Okie, updated with a jinja check.